provisions/roles/jenkins/slave/tasks/setup_worker.yml

```diff
@@ -19,48 +19,49 @@
   file: dest=/opt/cccp-service/beanstalk_worker state=directory
 
 - name: Build container-pipeline image
-  shell: docker build -t container-pipeline -f ./Dockerfile .
-  args:
-      chdir: /opt/cccp-service
+  sudo: yes
```

Comment:
Not really!

this is for user having only `sudo` access and provisioning happening with non-root account.